src/test/java/DefaultRecheckIgnoreJsTest.java

```diff
@@ -0,0 +1,27 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.event.Level;
+
+import de.retest.recheck.ignore.JSFilterImpl;
+import io.github.netmikey.logunit.api.LogCapturer;
+
+public class DefaultRecheckIgnoreJsTest {
+
+	@RegisterExtension
+	LogCapturer warningAndErrorLogs = LogCapturer.create() //
+			.forLevel( Level.WARN ) //
```

Comment:
Should this also be `Level.ERROR` to catch any potential errors in [JSFilterImpl.java#L44](https://github.com/retest/recheck/blob/master/src/main/java/de/retest/recheck/ignore/JSFilterImpl.java#L44)?

Reply:
Can we combine multiple levels? Because I tried it with a faulty file, and the test failed.

Reply:
I don't know the API in that regard. I would assume that ERROR would also catch WARN?

Reply:
You mean the other way around. If you e.g. define a filter in the log config, you define the _lowest_ level of interest.

Reply:
Yes!? 😕

Reply:
So I assume if we filter for WARN, everything _below_ will be filtered.